docs/customizing.rst

```diff
@@ -52,6 +52,266 @@ custom publish plugin UI.
 
 ----
 
+Step-by-Step Tutorials
+__________________________________
+
+Listed below are step-by-step tutorials to help you customize your publish
+workflows using publish plugins. See :ref:`publish-api` reference for more
+technical details of the Publish Plugin.
+
+Create a Publish Plugin
+****************************
```

Comment:
I don't know if this throws an error on Sphinx. Sometimes the bottom line should be equal in length. Did you try generate these docs?

```suggestion
Create a Publish Plugin
***********************
```

Reply:
Yeah, I generated the docs locally and it was fine. Also CI for doc validation has passed.